### ISSUE TITLE
gist is too wordy

### ISSUE THREAD

**Issue by lichtkind** [pierre-vigier/Perl6-Math-Matrix#38]
in mainline perl 6 gist shows only the first 100 elements of a list and I think we should do something in that spirit, like show only so much that fits on a scree page without line wrap

**Comment by pierre-vigier** [comment]
It makes total sense, but in that case do we need to create a kind of "pretty print" ?

**Comment by lichtkind** [comment]
what you mean by this? but generally yes :)

**Comment by pierre-vigier** [comment]
an easy way to print the full matrix to the console, for reading it entirely

**Comment by lichtkind** [comment]
for the case gist does not print the entire matrix?

**Comment by pierre-vigier** [comment]
yes, you were proposing display less value in the first comment in that discussion, i was referring to that, to have a second function, which would display the matrix entirely

**Comment by lichtkind** [comment]
allright sounds like a plan

**Comment by lichtkind** [comment]
after thinking about it normal string context is the pretty print so say ~$matrix should do the trick too and its less of an insular solution line say $matrix.pretty :)

**Comment by lichtkind** [comment]
I still look into it and given the function print and put have and what kind of output they expect I paddle back and stick to original plan to provide an pretty method

**Comment by lichtkind** [comment]
implemented  a few adjustments are still local but problem solved
